tests/e2e/e2e_gov_test.go

```diff
@@ -325,6 +325,123 @@ func (s *IntegrationTestSuite) testGovConstitutionAmendment() {
 	})
 }
 
+func (s *IntegrationTestSuite) testGovDynamicQuorum() {
+	s.Run("dynamic quorum change", func() {
+		// From the formulae in ADR-005
+		// pEma = (Q - Qmin) / (Qmax - Qmin)
+		// Participation = (PEma_t - (PEma_{t-1} * 0.8)/0.2
+		chainAAPIEndpoint := fmt.Sprintf("http://%s", s.valResources[s.chainA.id][0].GetHostPort("1317/tcp"))
+		params := s.queryGovParams(chainAAPIEndpoint, "tallying")
+		quorums := s.queryGovQuorums(chainAAPIEndpoint)
+		quorumRange := params.GetParams().QuorumRange
+		quorumMin := sdk.MustNewDecFromStr(quorumRange.Min)
+		quorumMax := sdk.MustNewDecFromStr(quorumRange.Max)
+		currentQuorum := sdk.MustNewDecFromStr(quorums.GetQuorum())
+		quorumPEma := (currentQuorum.Sub(quorumMin)).Quo(quorumMax.Sub(quorumMin))
+		paramsFeemarket := s.queryFeemarketParams(chainAAPIEndpoint)
+		oldAlpha := paramsFeemarket.Params.Alpha
+		paramsFeemarket.Params.Alpha = oldAlpha.Add(sdk.NewDec(1))
+		s.writeFeemarketParamChangeProposal(s.chainA, paramsFeemarket.Params)
```

Comment:
I would rather use a less specific proposal for this test, it's confusing to imply the feemarket in it. We should use a simple text proposal, but there is currently no test for a text proposal submission, right ?

Reply:
agreed, also this will make life easier for the renaming of the module, so we don't have leftover "feemarket" references